itur/models/itu840.py

```diff
@@ -3,52 +3,24 @@
 from __future__ import division
 from __future__ import print_function
 
-import os
 import numpy as np
+import os
 from astropy import units as u
 
 from itur.models.itu1144 import bilinear_2D_interpolator
-from itur.utils import (dataset_dir, prepare_input_array, prepare_output_array,
-                        prepare_quantity, memory, load_data_interpolator)
+from itur.utils import load_data, dataset_dir, prepare_input_array, \
```

Comment:
Why did you go back to the old method of loading data? using load_data_interpolator simplifies the code and makes it more maintainable.